Demo/src/main/java/com/braintreepayments/demo/PayPalNativeCheckoutRequestFactory.java

```diff
@@ -8,6 +8,7 @@
 import com.braintreepayments.api.PostalAddress;
 
 public class PayPalNativeCheckoutRequestFactory {
+    public static final String PAYPAL_NATIVE_CHECKOUT_CLIENT_ID = "NativeXOTest";
```

Comment:
should we name this `PAYPAL_NATIVE_CHECKOUT_MERCHANT_ID`?

Reply:
yeah i will switch to merchant